edx_rest_framework_extensions/auth/jwt/tests/test_middleware.py

```diff
@@ -182,6 +182,46 @@ class HasNoCondPermView(APIView):
         )
         self.assertIn(NotJwtRestrictedApplication, HasNoCondPermView.permission_classes)
 
+    def test_conditional_permissions_drf(self):
+        """
+        Make sure we handle ConditionalPermissions from rest_condition.
```

Comment:
This comment is the same as the one before. I think it should be modified accordingly.